### DISCUSSION TITLE
Intallation Issue

### DISCUSSION THREAD

**Discussion by huababua** [cryinkfly/Autodesk-Fusion-360-for-Linux#375] [General]
Hi,
first of all - thank you for all the work you put into this project.

Unfortunately I am stuck during the installation. It went well until I got the error message;
![image](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/assets/123359525/9c1f878c-9ce5-4aa0-aea7-ea7539ba369f)

After that there were tons of error messages in the terminal;
![image](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/assets/123359525/044729b4-e268-4b60-bfcf-1899ab3fafd8)

Any help is very welcome!

Thanks
Frank

**Comment by Aus-gez** [comment]
Hi @huababua I get that error message too, and all the subsequent "fixme" ones in the terminal. However if you dismiss or ignore the Microsoft Edge installer error, the installer should continue for quite a few minutes until eventually showing signs of positive progress.

Your terminal hasn't returned to the command prompt, so it's still in the middle of things.

What happens if you leave it for another 10 mins?

**Comment by huababua** [comment]
![image](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/assets/123359525/f655032c-b2c0-461c-bb55-b6027206016f)
![image](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/assets/123359525/f2cce931-1c6f-455a-8dfa-8f6503ee8630)

error log from terminal:
`
wine: could not open working directory L"unix\\home\\frank\\.fusion360\\bin\\", starting in the Windows directory.
wine: could not open working directory L"unix\\home\\frank\\.fusion360\\bin\\", starting in the Windows directory.
setProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2) failed: COM error 0x57  (Unknown error 0x057)
"C:/Program Files/Autodesk/webdeploy/production/b0c303e70bd97cfdc195adab65922cfeffcb363a/plugins"
EnableNonClientDpiScaling() failed for HWND 0x30078 (120) (Call not implemented.)
info:  Game: ADPClientService.exe
info:  DXVK: v2.3
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x2d4ad3eb0
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  Skipping CPU adapter: llvmpipe (LLVM 15.0.7, 256 bits)
info:  Intel(R) Graphics (RPL-U):
info:    Driver : Intel open-source Mesa driver 23.3.2
info:    Memory Heap[0]: 
info:      Size: 23821 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[1]: Property Flags = 0x7
info:      Memory Type[2]: Property Flags = 0xf
info:  Game: ADPClientService.exe
info:  DXVK: v2.3
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x2d4ad3eb0
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  Skipping CPU adapter: llvmpipe (LLVM 15.0.7, 256 bits)
info:  Intel(R) Graphics (RPL-U):
info:    Driver : Intel open-source Mesa driver 23.3.2
info:    Memory Heap[0]: 
info:      Size: 23821 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[1]: Property Flags = 0x7
info:      Memory Type[2]: Property Flags = 0xf
info:  Game: ADPClientService.exe
info:  DXVK: v2.3
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x2d4ad3eb0
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  Skipping CPU adapter: llvmpipe (LLVM 15.0.7, 256 bits)
info:  Intel(R) Graphics (RPL-U):
info:    Driver : Intel open-source Mesa driver 23.3.2
info:    Memory Heap[0]: 
info:      Size: 23821 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[1]: Property Flags = 0x7
info:      Memory Type[2]: Property Flags = 0xf
qt.webenginecontext:

GLImplementation: desktop
Surface Type: OpenGL
Surface Profile: CompatibilityProfile
Surface Version: 4.6
Using Default SG Backend: yes
Using Software Dynamic GL: no
Using Angle: no

Init Parameters:
  *  application-name Fusion360
  *  browser-subprocess-path C:\Program Files\Autodesk\webdeploy\production\b0c303e70bd97cfdc195adab65922cfeffcb363a\QtWebEngineProcess.
exe
  *  create-default-gl-context
  *  disable-d3d11
  *  disable-es3-gl-context
  *  disable-features DnsOverHttpsUpgrade,ConsolidatedMovementXY,InstalledApp,BackgroundFetch,WebOTP,WebPayments,WebUSB,PictureInPicture

  *  disable-gpu-rasterization
  *  disable-speech-api
  *  enable-features NetworkServiceInProcess,TracingServiceInProcess
  *  enable-threaded-compositing
  *  ignore-gpu-blocklist
  *  in-process-gpu
  *  log-severity disabled
  *  no-proxy-server
  *  use-gl desktop

[260:264:0109/094012.065:ERROR:dxva_video_decode_accelerator_win.cc(1441)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: 
Module not found. (0x7E)
[260:264:0109/094012.080:ERROR:dxva_video_decode_accelerator_win.cc(1449)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: Modu
le not found. (0x7E)
[260:264:0109/094012.177:ERROR:network_change_notifier_win.cc(228)] WSALookupServiceBegin failed with: 8
info:  Game: Fusion360.exe
info:  DXVK: v2.3
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x2d4ad3eb0
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  Skipping CPU adapter: llvmpipe (LLVM 15.0.7, 256 bits)
info:  Intel(R) Graphics (RPL-U):
info:    Driver : Intel open-source Mesa driver 23.3.2
info:    Memory Heap[0]: 
info:      Size: 23821 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[1]: Property Flags = 0x7
info:      Memory Type[2]: Property Flags = 0xf

DevTools listening on ws://127.0.0.1:9766/devtools/browser/4f04db30-fc26-4e53-a542-30e05ed05f01
info:  Game: Fusion360.exe
info:  DXVK: v2.3
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x2d4ad3eb0
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
warn:  Skipping CPU adapter: llvmpipe (LLVM 15.0.7, 256 bits)
info:  Intel(R) Graphics (RPL-U):
info:    Driver : Intel open-source Mesa driver 23.3.2
info:    Memory Heap[0]: 
info:      Size: 23821 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[1]: Property Flags = 0x7
info:      Memory Type[2]: Property Flags = 0xf
[01/09/24 09:41:53] error    failed to parse version string
[01/09/24 09:41:53] error    failed to get OS kernel version
[01/09/24 09:41:53] error    failed to parse version string
[01/09/24 09:41:53] error    failed to get Wininet version
[01/09/24 09:41:53] error    failed in-client roundtrip task. Microsoft.Diagnostics.Runtime.ClrDiagnosticsException: Could not load cras
h dump, HRESULT: 0x80004001
  at Microsoft.Diagnostics.Runtime.DbgEngDataReader..ctor (System.String dumpFile) [0x00065] in <a3988a563127448cbae66914fb7dc8dc>:0
  at Microsoft.Diagnostics.Runtime.DataTarget.LoadCrashDump (System.String fileName) [0x00000] in <a3988a563127448cbae66914fb7dc8dc>:0 
  at cer_component.dumpparser.DumpParser.Load (System.String dumpFile, System.String cacheFolder) [0x00000] in <75a6a7fb9b244c149fac5baf
42f9acd8>:0
  at cer_component.StartupApp+<>c__DisplayClass0_1.<Main>b__0 () [0x0002a] in <75a6a7fb9b244c149fac5baf42f9acd8>:0
created crash info xml C:\users\frank\AppData\Local\Autodesk\CER\742e123e936617c8d98115753092c2b020892902\1\dmpuserinfo.xml
`

**Comment by Aus-gez** [comment]
Before you last post I was going to say your issue looks like #350. However, now I'm not sure, sorry.

I had success a few weeks ago with the same version of wine.

Some basic troubleshooting:
* Have you installed your prerequisites? I recall I needed to install two requirements manually using `sudo apt install cabextract winbind`
* If either of the above wasn't present, delete your fusion 360 wineprefix folder and start the install again.

**Comment by huababua** [comment]
Is there a manual way of deinstalling everything - the "Fusion360 (Uninstall)" does exactly nothing after I have selected the wineprefix?

Thanks

**Comment by Aus-gez** [comment]
You can delete the `\home\frank\.fusion360` directory to delete everything the script previously created.
